INSTALL.rst

```diff
@@ -91,8 +91,3 @@ Access the record (web/rest)
 
    firefox http://localhost:5000/records/1
    curl -i -H "Accept: application/json" http://localhost:5000/api/records/1
-
-
-======
-ENJOY!
-======
```

Comment:
Hehe, I brought it back, but without the headline causing issues with table of contents.